go/cmd/vtctldclient/command/keyspaces.go

```diff
@@ -61,6 +61,24 @@ Otherwise, the keyspace must be empty (have no shards), or returns an error.`,
 		Args:                  cobra.ExactArgs(1),
 		RunE:                  commandDeleteKeyspace,
 	}
+	// DisableVtorcEmergencyReparent disables the use of EmergencyReparentShard in VTOrc recoveries for a given keyspace.
+	DisableVtorcEmergencyReparent = &cobra.Command{
+		Use:                   "DisableVtorcEmergencyReparent <keyspace>",
+		Short:                 "Disables the use of EmergencyReparentShard in VTOrc recoveries for a given keyspace.",
+		DisableFlagsInUseLine: true,
+		Aliases:               []string{"disablevtorcemergencyreparent"},
+		Args:                  cobra.ExactArgs(1),
+		RunE:                  commandDisableVtorcEmergencyReparent,
+	}
+	// EnableVtorcEmergencyReparent enables the use of EmergencyReparentShard in VTOrc recoveries for a given keyspace.
+	EnableVtorcEmergencyReparent = &cobra.Command{
+		Use:                   "EnableVtorcEmergencyReparent <keyspace>",
```

Comment:
We should only have 1 command and let it take a boolean to signify whether we are disabling or enabling.

Reply:
@GuptaManan100 makes sense, @deepthi had the same suggestion. I'll update the PR 👍 

We also agreed ERS disable per-shard would make sense. What do you think and should that be a 2nd PR or included here?

Reply:
Whichever you prefer.

Reply:
@GuptaManan100 ok, updated to support shards as well. The disabled state is stored in both keyspace and shard records, so that:
1. Disabling either is possible
2. If new shards are made in a keyspace with ERS disabled, the new shard will also be disabled